cmd/git-sync/main.go

```diff
@@ -174,8 +174,10 @@ var (
 	}, []string{"status"})
 )
 
-// Channels for ensuring hooks execute at least once before terminating due to GIT_SYNC_ONCE
-var execHookChannel, webHookChannel chan bool  // initialising as null to promptly catch logical errors and to avoid unneeded object creation
+// Channels for ensuring hooks execute at least once before terminating.
+// Should be nil if and only if corresponding hook is not defined and if initialised, will only every get written
```

Comment:
s/every/ever

also, please linewrap comments at 80

Reply:
Sounds good. I also need to double space sentences instead of single spacing.